GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml

```diff
@@ -148,4 +148,9 @@
     <a href="@Url.Action("ReturnsWithInvalidCompanyLink")" class="govuk-link">
         Submitted reports with invalid company link
     </a>
+</h2>
+<h2 class="govuk-heading-m">
+    <a href="@Url.Action("ReturnsWithInvalidOptedOutOfReportingPayQuartersValue")" class="govuk-link">
+        Submitted reports opted out of reporting pay quarters when not allowed
```

Comment:
Can you come up with a better description? Much appreciated 😄

Reply:
I didn't want to add "that" because it sounds like the report opted out, not the employer. But I'll use it anyway